tests/unit/src/test/scala/tests/codeactions/ImportMissingSymbolLspSuite.scala

```diff
@@ -17,6 +18,7 @@ class ImportMissingSymbolLspSuite
     s"""|${ImportMissingSymbol.title("Future", "scala.concurrent")}
         |${ImportMissingSymbol.title("Future", "java.util.concurrent")}
         |${CreateNewSymbol.title("Future")}
+        |${RewriteBracesParensCodeAction.toBraces}
```

Comment:
Why does the action pop up here? It shouldn't. Also in the other places in this test suite.

Reply:
I think it should. Look what is selected: part of the `Term.Apply`, to be specific you selected following
```json
                    "type": "Term.Apply",
                    "fun": {
                      "type": "Term.Select", 
                      "qual": {
                        "type": "Term.Name",
                        "value": "Future" <<< here
                      },
                      "name": {
                        "type": "Term.Name",
                        "value": "successful"
                      }
```
I think here it's a valid usage, tbh I'm more concerned about more ambiguous cases. Maybe, because of that, we should contribute this code action only when the selection range is equal to zero (just cursor, no selection at all)?